cli/lib/config.js

```diff
@@ -2,35 +2,37 @@
 
 module.exports = { load, save };
 
-const { promises: fs } = require('fs');
+const fs = require('fs');
 const toml = require('@iarna/toml');
 const home = require('user-home');
 const path = require('path');
 
 const errors = require('./errors');
 
 async function load(filename = path.join(home, '.entropicrc')) {
-  var content = null;
-  try {
-    content = await fs.readFile(filename, 'utf8');
-  } catch (e) {
-    if (e.code === 'ENOENT') {
-      return {};
-    }
-
-    throw new errors.CouldNotReadConfigFile(filename, e);
-  }
-
-  var parsed = null;
-  try {
-    parsed = toml.parse(content);
-  } catch (e) {
-    throw new errors.CouldNotParseConfigToml(filename, e);
-  }
+  var content = '';
+  var parsed = '';
+  await fs.readFile(filename, 'utf8', (e, data) => {    
```

Comment:
Why would you have a callback and `await` used together?

Reply:
That's true, doesn't make sense , shall i update it or will go with #186

Reply:
I'd wait for #186